From dd58f38c957429a862c1bf76d00b2b0da2ce06d6 Mon Sep 17 00:00:00 2001
From: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Date: Thu, 8 Apr 2021 09:20:44 +0200
Subject: [PATCH] mailbox: stm32-ipcc: Update build dependency

Align dependency with other stm32mp drivers, which use ARCH_STM32
not MACH_STM32MP157.

Signed-off-by: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Change-Id: Ie5a473ce4217a7f5888b7713b363338134360576
---
 drivers/mailbox/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/mailbox/Kconfig
+++ b/drivers/mailbox/Kconfig
@@ -233,7 +233,7 @@ config BCM_FLEXRM_MBOX
 
 config STM32_IPCC
 	tristate "STM32 IPCC Mailbox"
-	depends on MACH_STM32MP157 || COMPILE_TEST
+	depends on ARCH_STM32 || COMPILE_TEST
 	help
 	  Mailbox implementation for STMicroelectonics STM32 family chips
 	  with hardware for Inter-Processor Communication Controller (IPCC)
